include/BasicTypes.hpp

```diff
@@ -612,7 +612,8 @@ class NiSyncVector : public NiVectorBase<ValueType, SizeType> {
 	}
 
 	void GetStringRefs(std::vector<NiStringRef*>& refs) {
-		if constexpr (HasGetStringRefs<ValueType, std::set<NiStringRef*>>)
+		// why is this necessary?
+		if constexpr (HasGetStringRefs<ValueType, std::vector<NiStringRef*>>)
```

Comment:
Removed in ousnius. idk how I lost this change but my bad

Reply:
actually these are needed because SWIG instantiates a bunch of functions that fail to compile otherwise. I'll take your corrections.